zokrates_cli/examples/assign_to_element.zok

```diff
@@ -0,0 +1,11 @@
+def foo() -> u32:
+        return 0
```

Comment:
Not really necessary, but we should stick to "tabs" being 4 spaces for the sake of cleaner formatting and consistency

Reply:
Can we enforce this somehow? I don't care what we use as long as it's consistent and CI enforced

Reply:
Opened #901 for this and fixed this manually here